src/itemChoice/style.ts

```diff
@@ -42,7 +42,7 @@ export default css`
   }
 
   :global(.kirk-itemChoice + .kirk-itemChoice::before) {
-    content: '';
+    content : '';
```

Comment:
No space needed between the property and the colon

Reply:
I didn't touch the file, check the final diff